src/Packager/Packager/Packager.csproj

```diff
@@ -6,9 +6,9 @@
   </PropertyGroup>
 
   <Target Name="PostBuild" AfterTargets="PostBuildEvent">
-    <Exec Command="dotnet.exe restore $(ProjectDir)..\Bundle" />
+    <Exec Command="dotnet restore $(ProjectDir)../Bundle" />
   </Target>
   <Target Name="PostPostBuild" AfterTargets="PostBuild">
-    <Exec Command="&quot;$(MsBuildToolsPath)\msbuild.exe&quot; $(ProjectDir)..\Bundle\Bundle.csproj /t:publish /p:PublishDir=$(ProjectDir)..\..\Microsoft.IIS.Administration\plugins /p:Configuration=$(Configuration) /p:SignType=$(SignType) /p:SigningIdentity=$(SigningIdentity)" />
+    <Exec Command="dotnet msbuild $(ProjectDir)../Bundle/Bundle.csproj /t:publish /p:PublishDir=$(ProjectDir)../../Microsoft.IIS.Administration/plugins /p:Configuration=$(Configuration) /p:SignType=$(SignType) /p:SigningIdentity=$(SigningIdentity)" />
```

Comment:
This change breaks the signing scenario. Msbuild is only expected to run on Windows. If you would like you can change the behavior based on whether $(SignType) is defined.

Reply:
Thank you for your comment, I will split this post-build action to support both worlds.